charts/kubefed/charts/controllermanager/templates/kubefedconfig.yaml

```diff
@@ -24,12 +24,13 @@ spec:
 {{- if .Values.featureGates }}
   - name: PushReconciler
     configuration: {{ .Values.featureGates.PushReconciler | default "Enabled" | quote }}
-  - name: RawResourceStatusCollection
-    configuration: {{ .Values.featureGates.RawResourceStatusCollection | default "Disabled" | quote }}
   - name: SchedulerPreferences
     configuration: {{ .Values.featureGates.SchedulerPreferences | default "Enabled" | quote }}
   - name: CrossClusterServiceDiscovery
     configuration: {{ .Values.featureGates.CrossClusterServiceDiscovery | default "Disabled" | quote }}
   - name: FederatedIngress
     configuration: {{ .Values.featureGates.FederatedIngress | default "Disabled" | quote }}
+  # NOTE: Commented feature gate to fix https://github.com/kubernetes-sigs/kubefed/issues/1333
+  #- name: RawResourceStatusCollection
+  #  configuration: {{ .Values.featureGates.RawResourceStatusCollection | default "Disabled" | quote }}
```

Comment:
* Does this completely remove the effects of setting this feature gate?
* When can we enable this again?

Reply:
Generally, this won't disable the feature. With the current code, if you set it in values to true, the hook will enable it. I had to explicitly remove it here to avoid older webhook to fail and so the helm upgrade. But anyone could now upgrade from v0.5.0 to v0.6.0 enabling it.
It is more of a hidden it, rather than disabling it.

Reply:
Wdyt?